bio/pileup/pileup.go

```diff
@@ -60,12 +56,12 @@ type Pileup struct {
 	Quality       string   `json:"quality"`
 }
 
-// Parse parses a given Pileup file into an array of Pileup structs.
-func Parse(r io.Reader) ([]Pileup, error) {
-	// 32kB is a magic number often used by the Go stdlib for parsing. We multiply it by two.
-	const maxLineSize = 2 * 32 * 1024
-	parser := NewParser(r, maxLineSize)
-	return parser.ParseAll()
+// Header is a blank struct, needed for compatibility with bio parsers. It contains nothing.
+type Header struct{}
+
+// WriteTo is a blank function, needed for compatibility with bio parsers. It doesn't do anything.
```

Comment:
Why no writer?

Reply:
There is a writer, `WriteTo` (needed for interface compatibility). Pileup files don't have headers though, so in this case it doesn't do anything.